### ISSUE TITLE
Support subset of self-reporting questions to be one time only questions

### ISSUE THREAD

**Issue by qingqingliu** [safe-paths-contrib/covid-safe-paths#19]
Subset of self-reporting questions should be asked only once. User can change the answer to those questions from settings. When perform self reporting, answers to those questions will be retrieved and sent automatically. 

Please refer to [this doc](https://docs.google.com/document/d/1Q-37obNuIZz2jARz7dsPA6rBKJITolS62IJGp1l08g8/edit?ts=5e9794ba) for what are part of one time questions 

- [ ] Store answers to one time questions
- [ ] Allow user to change answers to those questions from settings
- [ ] Allow easy retrieval of those answers during self reporting at submission time

**Comment by kujenga** [comment]
This is the ticket that is what the design is waiting on, this is the target of the branching strategy worked on elsewhere.

Onboarding questions like "age range", etc. are one-time. Since we're not doing onboarding in initial release, need to integrate them with the flow.

The idea here is that questions answers would be stored on-device.

Question: Is this needed for MVP?

**Comment by kujenga** [comment]
Moving to the top of the icebox to represent low priority
